config/routes.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+  get 'post/new'
+  get 'post/create'
+  get 'post/index'
+  get 'static_pages/home'
+  root "post#index"
```

Comment:
As I said before. Controller name should be plural. So after changing the controller name, it  should be `posts#index`